dd-sdk-android-internal/src/main/java/com/datadog/android/internal/utils/ImageViewUtils.kt

```diff
@@ -4,17 +4,25 @@
  * Copyright 2016-Present Datadog, Inc.
  */
 
-package com.datadog.android.sessionreplay.internal.utils
+package com.datadog.android.internal.utils
 
 import android.graphics.Rect
 import android.graphics.drawable.Drawable
 import android.view.View
 import android.widget.ImageView
-import com.datadog.android.sessionreplay.internal.recorder.densityNormalized
-import com.datadog.android.sessionreplay.model.MobileSegment
 
-internal object ImageViewUtils {
-    internal fun resolveParentRectAbsPosition(view: View): Rect {
+/**
+ * A collection of view utility functions for resolving absolute
+ * positions, clipping bounds, and other useful data for
+ * image views operations.
+ */
+object ImageViewUtils {
+    /**
+     * Resolves the absolute position on the screen of the given [View].
+     * @param view: the [View].
```

Comment:
minor: no need to add `:` after the parameter name (otherwise, I guess, it may be considered as a part of the name in the rendered doc?), same applies to all similar lines in this file (and others)